libcxx/src/include/sso_allocator.h

```diff
@@ -34,7 +35,7 @@ class _LIBCPP_HIDDEN __sso_allocator<void, _Np>
 template <class _Tp, size_t _Np>
 class _LIBCPP_HIDDEN __sso_allocator
 {
-    typename aligned_storage<sizeof(_Tp) * _Np>::type buf_;
```

Comment:
This can change the object size & alignment, no?

`aligned_storage` may add padding bytes at the end, where this would not. And the alignment wasn't specified when using `aligned_storage`, so it may have been larger than what the type needed.

Are we certain this isn't ABI breaking?

Reply:
If you look carefully, this is inside `libcxx/src` and it's only used to initialize a library-internal vector of locales. So while this *might* be ABI changing, it shouldn't matter since that type was never part of our ABI surface!